examples/prometheus/README.md

```diff
@@ -0,0 +1,137 @@
+# Monitoring STUNner with Prometheus and Grafana
```

Comment:
Guys, you're gonna hate me, but I think this one is not really a tutorial (that is, this should not end up here: https://github.com/l7mp/stunner/tree/main/doc#tutorials), but much rather a user guide (i.e., it should go here: https://github.com/l7mp/stunner/tree/main/doc#user-guides). Tutorials show how to set up this or that WebRTC application with STUNner, while this guide should be useful with *any* of the demos to monitor STUNner.  

This means that we'll have to rephrase the text a bit: I recommend the following structure:
* Description (what metrics we expose, how monitoring data flows from STUNner to Prom/Grafana, etc., maybe a fig would be useful too + a disclaimer that we are in the process to add further metrics to the single metric we expose in the next releases) 
* Configuration (i.e, helm and the like)
* Setting up Prometheus
* Setting up Grafana